doc/userguide/lua/lua-functions.rst

```diff
@@ -1052,6 +1052,22 @@ You can use it to set string
          SCFlowvarSet(0, a, #a)
      end 
 
+
+.. note::
+
+   During detection, the following steps happen, in that ordem:
```

Comment:
what is this detection here? Whole detect engine? Just the script? It's not very clear to me.

Reply:
Okie, to try and verify, I tried to use what was described in the suricata-verify test. Should have pondered more about what I'd add here.